FIO/local-volumes/dbench-job-generator-local-volume-replica.sh

```diff
@@ -15,78 +18,33 @@ set -euo pipefail
 #  - jq in the PATH
 #
 
-# Define some colours for later
-RED='\033[0;31m'
-GREEN='\033[0;32m'
-NC='\033[0m' # No Color
-
 echo -e "${GREEN}Scenario: Local Volume with a replica${NC}"
 echo
 
-# Checking if jq is in the PATH
-if ! command -v jq &> /dev/null
-then
-    echo -e "${RED}jq could not be found. Please install jq and run the script again${NC}"
-    exit 1
-fi
+check_for_jq
 
 # Checking if StorageOS Cli is running as a pod, if not the script will deploy it
-CLI_VERSION="storageos/cli:v2.5.0"
-STOS_NS="kube-system"
-cli_pod=$(kubectl -n ${STOS_NS} get pod -lrun=cli --no-headers -ocustom-columns=_:.metadata.name)
-
-if [ "${cli_pod}" != "cli" ]
-then
-    echo -e "${RED}StorageOS CLI pod not found. Deploying now${NC}"
-
-    kubectl -n ${STOS_NS} run \
-    --image ${CLI_VERSION} \
-    --restart=Never                          \
-    --env STORAGEOS_ENDPOINTS=storageos:5705 \
-    --env STORAGEOS_USERNAME=$(kubectl -n ${STOS_NS} get secrets init-secret -oyaml | awk '/username:/ {print $2}' |  base64 --decode) \
-    --env STORAGEOS_PASSWORD=$(kubectl -n ${STOS_NS} get secrets init-secret -oyaml | awk '/password:/ {print $2}' |  base64 --decode) \
-    --command cli                            \
-    -- /bin/sh -c "while true; do sleep 999999; done"
-fi
-
-sleep 5
-SECONDS=0
-TIMEOUT=30
-while ! kubectl -n ${STOS_NS} get pod cli -otemplate="{{ .status.phase }}" 2>/dev/null| grep -q Running; do
-  pod_status=$(kubectl -n ${STOS_NS} get pod cli -otemplate="{{ .status.phase }}" 2>/dev/null)
-  if [ $SECONDS -gt $TIMEOUT ]; then
-      echo "The pod cli didn't start after $TIMEOUT seconds" 1>&2
-      echo -e "${RED}Pod: cli, is in ${pod_status}${NC} state."
-      exit 1
-  fi
-  sleep 5
-done
+cli=$(get_cli_pod)
+echo -e "${GREEN}CLI pod: ${cli}"
 
 # Get the node name and id where the volume will get provisioned and attached on
 # Using the StorageOS cli is guarantee that the node is running StorageOS
-node_details=$(kubectl -n ${STOS_NS} exec cli -- storageos describe nodes -ojson | jq -r '[.[0].labels."kubernetes.io/hostname",.[0].id]')
-local_node_name=$(echo $node_details | jq -r '.[0]')
-local_node_id=$(echo $node_details | jq -r '.[1]')
+nodes_and_ids=($(get_storageos_nodes))
+node_name=${nodes_and_ids[0]#*~}
+node_id=${nodes_and_ids[0]%~*}
```

Comment:
There is an inconsistency with these expansions, name and id are inverted when using the script with replica
```suggestion
node_name=$(echo ${nodes_and_ids[0]} | cut -d "~" -f1)
node_id=$(echo ${nodes_and_ids[0]} | cut -d "~" -f2)
```